### DISCUSSION TITLE
icons problem

### DISCUSSION THREAD

**Discussion by rraammiinn** [vuetifyjs/vuetify#18349] [Q&A]
why many icons (mdi-group for example) don't work ?

**Comment by KaelWD** [comment]
[Dunno](https://play.vuetifyjs.com/#eNqLrlYqLkrWdywo0CsrTVWyUrIpSc0tyEksSbWLyVNQsCnTTSwoADPBnOT8vJLEzLzUIqgQWDATKGqXm5Kpm16UX1pgow8VgWoCclF1gUQghtroI1mmVKujZKxnrGeiBKENLZViAedyMVU=)
